packages/ttypescript/src/PluginCreator.ts

```diff
@@ -124,7 +124,24 @@ export class PluginCreator {
         this.validateConfigs(configs);
     }
 
-    createTransformers(params: { program: ts.Program } | { ls: ts.LanguageService }) {
+    mergeTransformers(into: ts.CustomTransformers, source: ts.CustomTransformers | TransformerBasePlugin) {
+        const slice = <T>(input: T | T[]) => Array.isArray(input) ? input.slice() : [input]
+        if (source.before) {
+            if (into.before) into.before.push(...slice(source.before));
+            else into.before = slice(source.before);
+        }
+        if (source.after) {
+            if (into.after) into.after.push(...slice(source.after));
+            else into.after = slice(source.after);
+        }
+        if (source.afterDeclarations) {
+            if (into.afterDeclarations) into.afterDeclarations.push(...slice(source.afterDeclarations));
+            else into.afterDeclarations = slice(source.afterDeclarations);
+        }
+        return this
+    }
+
+    createTransformers(params: { program: ts.Program } | { ls: ts.LanguageService }, customTransformers?: ts.CustomTransformers) {
         const chain: {
             before: ts.TransformerFactory<ts.SourceFile>[];
```

Comment:
Set type here to TransformersList

Reply:
Why? it'll receive `ts.CustomTransformers` which is different from the `TransformersList` you want since this last one would REQUIRE properties to be present